silo-core/contracts/incentives/interfaces/ISiloIncentivesControllerGaugeLike.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.28;
+
+/// @dev Interface for the SiloIncentivesControllerGaugeLike contract
+/// @notice Implements function to make the silo incentives controller gauge like
+interface ISiloIncentivesControllerGaugeLike {
```

Comment:
if all this is required to add contract as gauge, then we should replace `IGaugeLike` with this interface, because now we have two interfaces and whith one is correct?

Reply:
Bot are correct. They have functions that are required for the components that use them.

Reply:
Removed `ISiloIncentivesControllerGaugeLike `. All methods moved into `IGaugeLike`